paddle/majel/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cc_library(majel SRCS place.cc)
+cc_library(majel SRCS place.cc allocation.cc)
```

Comment:
I find that cc_library does not support merge static libraries now. So,   ```cc_library(majel DEPS palce allocation)``` will be wrong.